packages/jsii-pacmak/lib/markdown.ts

```diff
@@ -20,7 +20,17 @@ export function md2rst(text: string) {
   }
 
   function textOf(node: commonmark.Node) {
-    return node.literal ?? '';
+    if (node.literal) {
+      if (
+        node.prev?.literal === '\\' && // following a \ (might need escaping)
+        !/^\\.*/.exec(node.literal) && // doesn't start with a \ (not being escaped)
+        node.prev.prev?.literal !== '\\' // not two \ in a row (not already escaped)
+      ) {
+        return `\\${node.literal}`;
+      }
```

Comment:
This isn't a perfect test, but seems unlikely to need expansion in practice.

Reply:
At least it's a start and it appears to be sufficient for the test... I don't see a reason to over-complicate this...